processor/memorylimiterprocessor/memorylimiter.go

```diff
@@ -100,3 +102,21 @@ func (p *memoryLimiterProcessor) processLogs(ctx context.Context, ld plog.Logs)
 	p.obsrep.accepted(ctx, numRecords, pipeline.SignalLogs)
 	return ld, nil
 }
+
+func (p *memoryLimiterProcessor) processProfiles(ctx context.Context, td pprofile.Profiles) (pprofile.Profiles, error) {
+	numProfiles := td.SampleCount()
+	if p.memlimiter.MustRefuse() {
+		// TODO: actually to be 100% sure that this is "refused" and not "dropped"
+		// 	it is necessary to check the pipeline to see if this is directly connected
+		// 	to a receiver (ie.: a receiver is on the call stack). For now it
+		// 	assumes that the pipeline is properly configured and a receiver is on the
+		// 	callstack and that the receiver will correctly retry the refused data again.
```

Comment:
tangent: I wonder whether this comment (which appears 4 times now) will ever be resolved.

Reply:
Especially without an issue. Maybe replace it with an issue is the first step.

Reply:
Created and added as a comment.